### ISSUE TITLE
My Switch doesnt Work anymore only black Screen

### ISSUE THREAD

**Issue by Kleinermann556** [StarDustCFW/Haku33#29]
It has only black Screen and the Home Button is shining
Pls Help Mr

**Comment by DigitalJockey** [comment]
I’m having the exact same issue. The fix I found for now is to wipe the system with Tegra Explorer and boot Atmosphere from TegraExplorer, but after you do that you’ll realize that the only thing erased was the Haku33 application from the switch folder. And the home button is still blinking the blue light.

**Comment by jorgermiranda** [comment]
Hi @Kleinermann556 @DigitalJockey! have you been able to fix it?

**Comment by DigitalJockey** [comment]
No, not yet and I've tried everything.  ---- Em ter.,16 jan. 2024 02:41:37 -0300  ***@***.***  escreveu ----
Hi @Kleinermann556 @DigitalJockey! have you been able to fix it?

**Comment by Kronos2308** [comment]
Try running tegra explorer open emmc system and delete the save located on 
System/save/8000000000000120
Then run atmosphere again 
And the save will be rebuilt

**Comment by jorgermiranda** [comment]
Should we run atmosphere from TegraExplorer by using the option "run atmosphere atmosphere/reboot_payload.bin" @Kronos2308 ?? Should we update atmosphere?

I was able to delete the save file you describe and by running atmosphere from that option I'm having a black screen to. I'm not able to run the original firmware and neither atmosphere.

Could you please help us with more detailed steps and/or any advise to have the console by using atmosphere back again?

Thanks in advance

**Comment by Kronos2308** [comment]
Yes just load atmosphere after deleting thise files to rebuild the system save and nca data base

**Comment by DigitalJockey** [comment]
What deleting the file is trying to achieve, @Kronos2308? Is it to make Haku33 work or to restore the device to a pre-jailbreak state? Because I've deleted the file and Haku33 is still not working and Atmosphere is still installed on the device, só I'm a little lost.

**Comment by Kronos2308** [comment]
Come to my discord

**Comment by DigitalJockey** [comment]
What's your Server address? ---- Em seg, 22 jan 2024 19:32:13 -0300  ***@***.***  escreveu ----
Come to my discord

**Comment by Kronos2308** [comment]
https://discord.gg/kscctYj

**Comment by xXMOOXx** [comment]
I had the same problem. Updating Atmosphere did it for me, you should do that first. Then to fix it, i had to start (run) the TegraExplorer.bin, then navigate to "browse emmc" -> "system" -> "saves" and delete all the files there. Then go back to the first folder and run atmosphere from there (im not sure how it was called but something like "run atmosphere.bin/payload". But remember to first update your Atmosphere to the latest! Then the console started and was factory reset.

**Comment by ymaniz09** [comment]
@xXMOOXx Thanks!

It worked like a charm

**Comment by b0rd1** [comment]
> TegraExplorer.bin

Thank you, it works perfectly!

**Comment by Kronos2308** [comment]
We found 3 solution for the problem
1 just run atmosphere on Sysnand 
2 delete system:/saves/8000000000000120 using tegra explorer.bin
And then run atmosphere to rebuild db
Or 
3 delete system:/saves completely using tegra explorer.bin an the run atmosphere (this will make a full factory restart)

I I'll leave this open until whoever created it sees fit.

**Comment by ChatSlayer** [comment]
This happened to me as well, restored with backup (tried solutions 1 and 2, didn't want to factory reset, so I just used backup). What is the best firmware version for this?

**Comment by Kronos2308** [comment]
Use only the 2 option to not make a factory reset

**Comment by DigitalJockey** [comment]
> This happened to me as well, restored with backup (tried solutions 1 and 2, didn't want to factory reset, so I just used backup). What is the best firmware version for this?

Same here. Tried both options but the Atmosphere apps are still running when I boot it using the dongle like nothing happened.

Regular boot everything seems to be starting from scratch. I set up everything as usual but when using the dongle all the Atmosphere apps are still there. 

I don't have a backup and don't know what to do anymore.

**Comment by RJLeafar** [comment]
Buenas 

Tenia CFW en sysnand ( nada de emunand ) con la version de FW 13.2.1 y 16 efuses quemados e hice backup que evidentemente esta comprometido.
use Haku33 version 7.0 y siempre acababa aparentemente bien pero al reiniciar aparecia el logo horizontal de nintendo, pasaba a negro y ya nada... Ni arrancando directamente, ni desde hetake stock ni hetake CFV... el resultado siempre era el mismo.
restaure el backup y lo intente 4 veces mas... con y sin modo avion, y todo lo que lei.
al final probe con Haku33 version 4.6 que es especifica para la version 13.2.1 y funcionó. despues hice un restablecimiento a fabrica, esta vez desde el menu de la consola, y todo bien.

empece a configurar la consola y para vincular el usuario me pidio actualizar, meti las calves del wifi y actualicé pensando que actualizaria a las 18.0.1 ... Se actualizó y ya no enciendo mas... mismo problema que antes... Logo horizontal de Nintendo y a negro.

he pasado hekate la he puesto en autorcm y he visto que ha pasado a version 17.0.0+ con 19 efuses quemados.

despues de leer de todo, he entrado en Tegraexplorer me he metido en emmc, system, saves y hay muchisimos archivos ahi, asi que me he salido sin hacer nada.

alguna idea ?

he entrado con Hekate 611 y Atmos 170

gracias.


SOLUCIONADO.

solo tuve que entrar en atmosphere con fusee.bin
